contracts/contracts/FillManager.sol

```diff
@@ -7,9 +7,19 @@ import "OpenZeppelin/openzeppelin-contracts@4.5.0/contracts/access/Ownable.sol";
 import "../interfaces/IProofSubmitter.sol";
 import "./BeamerUtils.sol";
 
+/// The fill manager.
+///
+/// This contract is responsible for keeping track of filled requests. In addition to allowing
+/// agents to (eventually) prove that they filled requests, it also allows anyone to invalidate
+/// a claim that a request was filled.
```

Comment:
More accurately, it would invalidate a fill hash (or a fill id in combination with a request (hash) , respectively)

Reply:
Right, but this is high-level description of the contract, we're not really concerned about details on that level.